libraries/SPISlave/src/SPISlave.cpp

```diff
@@ -72,6 +72,14 @@ void SPISlaveClass::begin()
     hspi_slave_onStatusSent(&_s_status_tx);
     hspi_slave_begin(4, this);
 }
+void SPISlaveClass::end()
+{
+    hspi_slave_onData(NULL);
```

Comment:
Please use nullptr instead of NULL

Reply:
the hspi_slave_* functions are C functions called from C++. should nullptr  be used?

Reply:
Yes. NULL is a #define 0 in C, so an int that gets cast into a 32bit ptr, and in C++ it is a #define 0x00 that is cast into a void*, while nullptr is a C++ builtin that is really a null ptr. It's a type and/or style thing. As a rule of thumb, use of nullptr is considered better in C++ code. You can google nullptr vs. NULL to read about the C++ decision.